dsl/camel-jbang/camel-jbang-core/pom.xml

```diff
@@ -151,6 +151,12 @@
             <artifactId>plexus-xml</artifactId>
         </dependency>
 
+        <dependency>
```

Comment:
Yeah we need to not have this dependency. 

You can dynamic download this JAR via that camel maven thing, and we can maybe try set its scope = provided so camel run will not use it normally. And then you can still compile and use its API just that you have downloaded the JAR first. 

Otherwise we can also move the command to its own camel jbnag plugin then the dependency can be there (just k8s has kube client JAR etc). But then users need to install this plugin first before they can use it.

Reply:
I removed the dependency, it is still possible to run everything as expected via reflection (thanks to the jar download), I'll add some smoke tests for the reflection part.